packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -69,16 +69,32 @@ export class SelectionZone extends BaseComponent<ISelectionZoneProps, {}> {
   private _shouldHandleFocusTimeoutId: number | undefined;
   private _isTouch: boolean;
   private _isTouchTimeoutId: number | undefined;
+  private _findScrollParentAnimationFrameId: number | undefined;
 
   public componentDidMount(): void {
     const win = getWindow(this._root.current);
-    const scrollElement = findScrollableParent(this._root.current);
 
     // Track the latest modifier keys globally.
     this._events.on(win, 'keydown, keyup', this._updateModifiers, true);
-    this._events.on(scrollElement, 'click', this._tryClearOnEmptyClick);
     this._events.on(document.body, 'touchstart', this._onTouchStartCapture, true);
     this._events.on(document.body, 'touchend', this._onTouchStartCapture, true);
+
+    const animationFrameWindow = win || window;
+    this._findScrollParentAnimationFrameId = animationFrameWindow.requestAnimationFrame(() => {
+      const scrollParent = findScrollableParent(this._root.current);
```

Comment:
Interesting.. I'm wondering if we should have the same consideration for the `findScrollableParent` in `List`.

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/List/List.tsx#L295